python3: can't open file '/data/data/com.termux/files/home/osanvault-africa/ai/property_updater.py': [Errno 2] No such file or directory
